docs/insights/api.md

```diff
@@ -0,0 +1,511 @@
+
+### EnvironmentGroups Properties
+| Property |  | Description |
+| -------- | -------- | ----------- |
+| Name | Required | The name of the environment group.  Must be unique per report |
+| Environments | Required | An array of the ids of the environments to include in the group |
+
+
+## Get a specific Insights report
+GET: api/{spaceId}/insights/reports/{reportId}
+Retrieves an existing Insights report.
+
+### Parameters
+| Name | Description |
+| ---- | ----------- |
+| spaceId | Id of the space containing the report |
+| reportId | Id of the report to retrieve |
+
+### Sample response
+```json
+{
+  "Id": "InsightsReports-1",
+  "SpaceId": "Spaces-1",
+  "Name": "High Volume Report",
+  "Description": "High volume tenanted deployments report",
+  "ProjectGroupIds": ["ProjectGroups-1", "ProjectGroups-2"],
+  "ProjectIds": ["Projects-1", "Projects-2"],
+  "TenantIds": ["Tenants-1", "Tenants-2", "Tenants-3"],
+  "TenantTags": [
+    "Deployment Group/Group A",
+    "Deployment Group/Group B",
+    "Deployment Group/Group C",
+    "Location/Australia",
+    "Location/Asia",
+    "Location/Europe",
+    "Location/US"
+  ],
+  "TenantMode": "TenantedAndUntenanted",
+  "EnvironmentGroups": [
+    {
+      "Name": "Test",
+      "Environments": ["Environments-1", "Environments-2"]
+    },
+    {
+      "Name": "Internal",
+      "Environments": ["Environments-3", "Environments-4"]
+    },
+    {
+      "Name": "Production",
+      "Environments": ["Environments-5", "Environments-6"]
+    }
+  ],
+  "TimeZone": "UTC",
+  "IconId": null,
+  "IconColor": null,
+  "Links": {
+    "Self": "/api/Spaces-1/insights/reports/InsightsReports-1",
+    "Logo": "/api/Spaces-1/insights/reports/InsightsReports-1/logo"
+  }
+}
+```
+
+### Properties
+| Property | Description |
+| -------- | ----------- |
+| Id       | The id of the Insights report | 
+| SpaceId  | The id of the space containing the report |
+| Name     | The name of the report | 
+| Description     | The description of the report | 
+| ProjectGroupIds | An array of the project group ids included in the report | 
+| ProjectIds | An array of the project ids included in the report | 
+| TenantIds  | An array of the tenant ids included in the report | 
+| TenantTags | An array of the tenant tags associated with this report | 
+| TenantMode | Indicates what types of deployments contributed to this report. May be TenantedAndUntenanted, Tenanted, or Untenanted |
+| EnvironmentGroups | An array of the environment groups included in this report | 
+| TimeZone  | The timezone used in the report, in Windows timezone format | 
+| IconId    | A [Font Awesome](https://fontawesome.com) icon name | 
+| IconColor | The icon color in hex format (example: '#0D80D8') | 
+| Links     | Links to other resources.  See [REST API Links](https://octopus.com/docs/octopus-rest-api#api-links) |
+
+## Update an Insights report
+PUT: api/{spaceId}/insights/reports/{reportId}
```

Comment:
I don't know if the leading slash should be added. I checked my API, and it included one
```suggestion
PUT: `/api/{spaceId}/insights/reports/{reportId}`
```